### PR TITLE
Release 3.2.1

### PR DESCRIPTION
**Pull request by rprouse**
This PR is for reviewing the changes for the 3.2.1 release only, **do not merge**. I will merge any changes back into master using a branch and pull request off of this branch.

Please review Changes.txt and ensure unintended changes are not included. I am preparing the build tonight, but will finish testing and do the release in the morning (April 19) assuming there are no issues.

### CODE REVIEW

**Comment by rprouse** [comment]
Note - I am not planning on doing a CF release for this point release because the few serious issues that prompted this release do not seem to apply to CF and nobody from the CF community has raised issues except for one that was not fixed. If I am mistaken, please comment.

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,52 @@
...
+ * 1247	Potential memory issue
+ * 1231	Build may silently fail some tests
+ * 1208 Apartment on assembly level broken
+ * 1161 NUnit3-Console should disallow the combination of --inprocess and --x86, giving an error message
+ * 1141 Explicit Tests get run when using --where with some filters
+ *  773	Upgrade Travis CI from Legacy Infrastructure
+ *  640 Keep CF Build (and other future builds) in Sync
+ *  332 Add CF to the Appveyor CI build
+
+NUnit 3.2 - March 5, 2016
```
Comment (CharliePoole) [inline CHANGES.txt]: If this is too picky, just ignore me, but earlier releases had the resolved issues in ascending order.

**Comment by CharliePoole** [comment]
Aside from my one comment, this looks good.

**Comment by rprouse** [comment]
It looks like I have some tabs in there too. Thanks for looking, I will fix.

**Comment by rprouse** [comment]
3.2.1 has been released, closing this PR as it is no longer needed.

**Pull request closed without merging**